test/main_test.js

```diff
@@ -9,9 +9,17 @@ import { run } from '../js/main';
 
 // The DOM expected by the run method of main.js
 const chart = '<div class="vizr-container"><div id="vizr-instructions"/><div class="vizr-charts"/></div>';
+const endpointBase = 'http://localhost/redcap/api/?type=module&prefix=vizr&page=';
```

Comment:
Do we really want localhost hard-coded? Maybe ok since this is a test - just not sure how it plays with CI.